maap/Secrets.py

```diff
@@ -0,0 +1,134 @@
+import requests
+import logging
+import json
+from maap.utils import endpoints
+from maap.utils import requests_utils
+from maap.utils import endpoints
+
+logger = logging.getLogger(__name__)
+
+class Secrets:
+    """
+    Functions used for member secrets API interfacing
+    """
+    def __init__(self, member_endpoint, api_header):
+        self._api_header = api_header
+        self._members_endpoint = f"{member_endpoint}/{endpoints.MEMBERS_SECRETS}"
+
+
+    def get_secrets(self):
+        """
+        Returns a list of secrets for a given user.
+
+        Returns:
+            list: Secret names for a given user.
+        """
+        try:
+            response = requests.get(
+                url = self._members_endpoint,
+                headers=self._api_header
+            )
+            logger.debug(response.text)
+            return json.loads(response.text)
+
+        except Exception as e:
+            raise(f"Error retrieving secrets: {e}")
+
+
+    def get_secret(self, secret_name=None):
+        """
+        Returns secret value for provided secret name.
+
+        Args:
+            secret_name (str, required): Secret name.
+
+        Returns:
+            string: Secret value.
+
+        Raises:
+            ValueError: If secret name is not provided.
+        """
+        if secret_name is None:
+            raise ValueError("Failed to get secret value. Please provide secret name.")
```

Comment:
```suggestion
            raise ValueError("Secret name parameter cannot be None.")
```
Updated the error message since we are not trying to issue a `GET` yet.